### PR TITLE
oneapmci tutorial

### PR DESCRIPTION
**Pull request by yuanbohan**
@loddit @zjhmale cc @xtang @刘征

### CODE REVIEW

**Comment by yuanbohan** [comment]
@loddit @zjhmale poke

**Comment by loddit** [comment]
LGTM

File: robots/oneapmci.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+## 进入 OneAPM Cloud Insight 界面
+
+在 Cloud Insight 界面点击「设置」，点击「配置平台服务」并选择「BearyChat」
+
+![](/images/tutorial/oneapmci_add_integration.png)
+
+## 设置「Webhook」
+
+将 BearyChat 中生成的 Webhook 地址填入「Hook 地址」中并保存
```
Comment (zjhmale) [inline robots/oneapmci.md]: 这里不应该是`{{ Webhook }}`么?

Comment (yuanbohan) [inline robots/oneapmci.md]: OK

**Comment by yuanbohan** [comment]
@zjhmale poke

**Comment by zjhmale** [comment]
LGTM

**Pull request merged**